CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Oneview Features supported
 - oneview_server_hardware
 - oneview_server_hardware_type
+- oneview_interconnect
```

Comment:
Please arrange the resources name alphabetically

Reply:
done